<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtFilterDialog</class>
 <widget class="QDialog" name="qtFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>91</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Filter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qtClient.qrc">
    <normaloff>:/Resources/icon.png</normaloff>:/Resources/icon.png</iconset>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>471</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="filesBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Match Files</string>
   </property>
   <property name="icon">
    <iconset resource="qtClient.qrc">
     <normaloff>:/Resources/file.png</normaloff>:/Resources/file.png</iconset>
   </property>
  </widget>
  <widget class="QCheckBox" name="directoriesBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Match Directories</string>
   </property>
   <property name="icon">
    <iconset resource="qtClient.qrc">
     <normaloff>:/Resources/directory.png</normaloff>:/Resources/directory.png</iconset>
   </property>
  </widget>
  <widget class="QComboBox" name="typeBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Name</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Extension</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Full Name</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Path</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Name (regex)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Extension (regex)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Full Name (regex)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Path (regex)</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valueEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Value</string>
   </property>
  </widget>
  <widget class="QToolButton" name="browseButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;u&gt;Path&lt;/u&gt;: documents/list.txt | &lt;u&gt;Name&lt;/u&gt;: list | &lt;u&gt;Extension&lt;/u&gt;: txt | &lt;u&gt;Full Name&lt;/u&gt;: list.txt</string>
   </property>
  </widget>
  <widget class="QLabel" name="sendLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>sending request...</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="qtClient.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>qtFilterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>qtFilterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
